From 54ea2d2838bb798b395e4ec7094c87408035a619 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 22 Jul 2013 16:59:37 +0800
Subject: [PATCH 457/486] ENGR00272043 Add HWC into i.MX6SL

Take the FB device not have fsl_disp_dev_property attribution as LDB FB device

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 mx6/hwcomposer/hwc_display.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/mx6/hwcomposer/hwc_display.cpp b/mx6/hwcomposer/hwc_display.cpp
index 51baa3d..f4ba655 100644
--- a/mx6/hwcomposer/hwc_display.cpp
+++ b/mx6/hwcomposer/hwc_display.cpp
@@ -82,10 +82,10 @@ static int hwc_judge_display_state(struct hwc_context_t* ctx)
         snprintf(tmp, sizeof(tmp), "%s/fsl_disp_dev_property", fb_path);
         if (!(fp = fopen(tmp, "r"))) {
             ALOGI("open %s failed", tmp);
-            continue;
+            //continue;
             //make default type to ldb.
-            //pInfo->type = HWC_DISPLAY_LDB;
-            //pInfo->connected = true;
+            pInfo->type = HWC_DISPLAY_LDB;
+            pInfo->connected = true;
         }
         else {
             memset(value, 0, sizeof(value));
-- 
1.8.0

